restaurants/views.py

```diff
@@ -1,3 +1,84 @@
-from django.shortcuts import render
+from django.views           import View
+from django.http            import JsonResponse
+from django.db.models       import Avg, Q
+from django.core.exceptions import FieldError
 
-# Create your views here.
+from restaurants.models     import Restaurant
+from reviews.models         import Review, ReviewImage
+from users.utils            import login_decorator
+
+class RestaurantDetailView(View):
+    def get(self, request, restaurant_id):
+        try:
+            restaurant         = Restaurant.objects.get(id = restaurant_id)
+            reviews            = restaurant.review_set.all()
+            is_wished          = restaurant.wishlist_set.exists()
+            menus              = restaurant.menu_set.all()
+            rating_num         = restaurant.review_set.all().aggregate(rating = Avg('rating'))['rating']
+            
+            results = []
+            results.append({
+                "id"             : restaurant.id,
+                "name"           : restaurant.name,
+                "rating"         : rating_num,
+                "restaurant_img" : reviews.last().reviewimage_set.last().image,
+                "address"        : restaurant.address,
+                "phone_number"   : restaurant.phone_number,
+                "category"       : restaurant.category.name,
+                "location"       : restaurant.location.area,
+                "serving_price"  : restaurant.serving_price.price,
+                "menus"          : [{   
+                                        "item"       : menu.item, 
+                                        "item_price" : menu.item_price
+                                    } for menu in menus],
+                "is_wished"      : is_wished,
+                "review"         : [{
+                                        "review_id"  : review.id,
+                                        "user_id"    : review.user.id,
+                                        "user_name"  : review.user.nickname,
+                                        "description": review.description,
+                                        "rating"     : review.rating,
+                                        "created_at" : review.created_at,
+                                        "images url" : review.reviewimage_set.last().image,
+                } for review in reviews]
```

Comment:
리뷰 목록을 반환하므로 `reviews`로 key 값을 수정해주세요~
```suggestion
                "reviews"         : [{
                    "id"  : review.id,
                    "user" : {
                        "id" : review.user.id,
                        "name"  : review.user.nickname,
                    },
                    "description": review.description,
                    "rating"     : review.rating,
                    "created_at" : review.created_at,
                    "images url" : review.reviewimage_set.last().image,
                } for review in restaurant.review_set.all()]
```

Reply:
인덴트도 맞춰주세요!